src/fread.c

```diff
@@ -2745,72 +2727,72 @@ int freadMain(freadMainArgs _args) {
     }
   }
   setFinalNrow(DTi);
 
-  if (headPos<eof && DTi<nrowLimit) {
+  if (headPos < eof && DTi < nrowLimit) {
     ch = headPos;
-    while (ch<eof && isspace(*ch)) ch++;
-    if (ch==eof) {
+    while (ch < eof && isspace(*ch)) ch++;
+    if (ch == eof) {
       // whitespace at the end of the file is always skipped ok
     } else {
       const char *skippedFooter = ENC2NATIVE(ch);
       // detect if it's a single line footer. Commonly the row count from SQL queries.
-      while (ch<eof && *ch!='\n' && *ch!='\r') ch++;
-      while (ch<eof && isspace(*ch)) ch++;
-      if (ch==eof) {
-        DTWARN(_("Discarded single-line footer: <<%s>>"), strlim(skippedFooter,500));
+      while (ch < eof && *ch != '\n' && *ch != '\r') ch++;
+      while (ch < eof && isspace(*ch)) ch++;
+      if (ch == eof) {
+        DTWARN(_("Discarded single-line footer: <<%s>>"), strlim(skippedFooter, 500));
       }
       else {
         ch = headPos;
         int tt = countfields(&ch);
-        if (fill>0) {
+        if (fill > 0) {
           DTWARN(_("Stopped early on line %"PRId64". Expected %d fields but found %d. Consider fill=%d or even more based on your knowledge of the input file. Use fill=Inf for reading the whole file for detecting the number of fields. First discarded non-empty line: <<%s>>"),
-          DTi+row1line, ncol, tt, tt, strlim(skippedFooter,500));
+          DTi+row1line, ncol, tt, tt, strlim(skippedFooter, 500));
         } else {
           DTWARN(_("Stopped early on line %"PRId64". Expected %d fields but found %d. Consider fill=TRUE. First discarded non-empty line: <<%s>>"),
-          DTi+row1line, ncol, tt, strlim(skippedFooter,500));
+          DTi+row1line, ncol, tt, strlim(skippedFooter, 500));
         }
       }
     }
   }
-  if (quoteRuleBumpedCh!=NULL && quoteRuleBumpedCh<headPos) {
+  if (quoteRuleBumpedCh != NULL && quoteRuleBumpedCh<headPos) {
     DTWARN(_("Found and resolved improper quoting out-of-sample. First healed line %"PRId64": <<%s>>. If the fields are not quoted (e.g. field separator does not appear within any field), try quote=\"\" to avoid this warning."), quoteRuleBumpedLine, strlim(quoteRuleBumpedCh, 500));
   }
 
   if (verbose) {
     DTPRINT("=============================\n"); // # notranslate
-    if (tTot<0.000001) tTot=0.000001;  // to avoid nan% output in some trivially small tests where tot==0.000s
-    DTPRINT(_("%8.3fs (%3.0f%%) Memory map %.3fGB file\n"), tMap-t0, 100.0*(tMap-t0)/tTot, 1.0*fileSize/(1024*1024*1024));
-    DTPRINT(_("%8.3fs (%3.0f%%) sep="), tLayout-tMap, 100.0*(tLayout-tMap)/tTot);
-      DTPRINT(sep=='\t' ? "'\\t'" : (sep=='\n' ? "'\\n'" : "'%c'"), sep); // # notranslate
+    if (tTot < 0.000001) tTot = 0.000001;  // to avoid nan% output in some trivially small tests where tot==0.000s
+    DTPRINT(_("%8.3fs (%3.0f%%) Memory map %.3fGB file\n"), tMap - t0, 100.0 * (tMap - t0) / tTot, 1.0 * fileSize / (1024 * 1024 * 1024));
```

Comment:
Similar pedantic suggestion

```suggestion
    DTPRINT(_("%8.3fs (%3.0f%%) Memory map %.3fGiB file\n"), tMap - t0, 100.0 * (tMap - t0) / tTot, 1.0 * fileSize / (1024 * 1024 * 1024));
```